src/test/java/org/opentripplanner/apis/transmodel/mapping/TripRequestMapperTest.java

```diff
@@ -402,9 +402,14 @@ public void testExplicitModes() {
     assertEquals(StreetMode.WALK, req.journey().transfer().mode());
   }
 
+  /**
+   * This tests that both the new parameter name 'transferSlack` and the deprecated one
+   * 'minimumTransferTime' (for backwards compatibility) are correctly mapped to the internal
+   * transfer slack as a duration.
+   */
   @ParameterizedTest
   @ValueSource(strings = { "transferSlack", "minimumTransferTime" })
-  public void testTransferSlack(String name) {
+  public void testBackwardsCompatibleTransferSlack(String name) {
     Map<String, Object> arguments = Map.of(name, 101);
     var req = TripRequestMapper.createRequest(executionContext(arguments));
     assertEquals(Duration.ofSeconds(101), req.preferences().transfer().slack());
```

Comment:
I would have written this like the suggestion below witch I think is fare more readable. The reason why this test is confusing is that the variable arguments is not used to provide different test-cases, but alias/old name of the property. Then, if you need to extend the test-cases here it become very ugly. It also test two different things - we are not strict about that, but to some it might be confusing. You could also split it in 3 methods => no need for any comment, just clear names. 
```
@Test
public void testTransferSlack() {
    var d101s = Duration.ofSeconds(101);
    var req = TripRequestMapper.createRequest(executionContext(Map.of("transferSlack", 101)));
    assertEquals(d101s, req.preferences().transfer().slack());

    assertThrows(..., () ->
      TripRequestMapper.createRequest(executionContext(Map.of("transferSlack", <invalid value>)));
    );
    
    // Test the deprecated  'minimumTransferTime' for backwards compatibility
    req = TripRequestMapper.createRequest(executionContext(Map.of("transferSlack", 101)));
    assertEquals(d101s, req.preferences().transfer().slack());
}  
```